cypress-tests/cypress/e2e/v2/projectBasics.cy.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  getRandomString,
+  validateLoginV2,
+} from "../../support/commands/general";
+import { ProjectIdentifierV2 } from "../../support/types/project.types";
+import { User } from "../../support/types/user.types";
+import {
+  deleteProjectFromAPIV2,
+  getProjectByNamespaceAPIV2,
+} from "../../support/utils/projectsV2.utils";
+
+const sessionId = ["projectBasics", getRandomString()];
+
+beforeEach(() => {
+  // Restore the session (login)
+  cy.session(
+    sessionId,
+    () => {
+      cy.robustLogin();
+    },
+    validateLoginV2
+  );
+});
+
+describe("Project - create, edit and delete", () => {
+  // Define some project details
+  const projectNameRandomPart = getRandomString();
+  const projectName = `project/$test-${projectNameRandomPart}`;
+  const projectPath = `project-test-${projectNameRandomPart}`;
+  const projectDescription = "This is a test project from Cypress";
+  const projectIdentifier: ProjectIdentifierV2 = {
+    slug: projectPath,
+    id: null,
+    namespace: null,
+  };
+
+  // Cleanup the project after the test -- useful on failure
+  after(() => {
+    getProjectByNamespaceAPIV2(projectIdentifier).then((response) => {
+      if (response.status === 200) {
+        projectIdentifier.id = response.body.id;
+        projectIdentifier.namespace = response.body.namespace;
+        deleteProjectFromAPIV2(projectIdentifier);
+      }
+    });
+  });
+
+  it("Project - create, edit and delete", () => {
+    // Create a new project
+    cy.visit("/v2");
+    cy.getUserData().then((user: User) => {
+      const username = user.username;
+      cy.get("#plus-dropdown").should("exist").click();
+      cy.getDataCy("navbar-project-new").click();
+      cy.location("hash").should("eq", "#create-project");
+      cy.getDataCy("new-project-modal").should("exist");
+      cy.getDataCy("project-name-input").type(projectName);
+      cy.getDataCy("project-slug-toggle").click();
+      cy.getDataCy("project-slug-input").should("have.value", projectPath);
+      cy.getDataCy("project-visibility-public").click();
+      cy.getDataCy("project-description-input").type(projectDescription);
+      cy.getDataCy("new-project-modal").contains(
+        `The URL for this project will be renkulab.io/v2/projects/${username}/${projectPath}`
+      );
+      cy.getDataCy("project-create-button").click();
+      cy.location("pathname").should(
+        "eq",
+        `/v2/projects/${username}/${projectPath}`
+      );
```

Comment:
Instead of testing this, I think it is more important to test that the new page is the newly created project page, e.g. it should contain the project title and the project description.

Reply:
Good point, I'll change it!
Similar to https://github.com/SwissDataScienceCenter/renku/pull/3881/files#r1933949049

Reply:
fixed in 2f9a1256175006ed30996b31ee8ed90d95f90617